monai/handlers/iteration_metric.py

```diff
@@ -41,9 +47,11 @@ def __init__(
         metric_fn: Callable,
         output_transform: Callable = lambda x: x,
         device: Optional[torch.device] = None,
+        save_details: bool = False,
```

Comment:
I think this could default to True? the cost is tiny

Reply:
OK, updated in the latest commit.
Thanks.